pkg/helpers/utils.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+func close(c io.Closer) {
+	err := c.Close()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
If i summarize we should probably not use `defer` but directly deal with the possible error?

Reply:
1. we should definitely use `defer` (I didn't read anything against it)
2. we should **return** the error produced during the deferred statement (rather than log and exit)

This is just my interpretation of what I read on the subject.
And not necessarily hard rules.